src/javascript/Form/GeometryForm.js

```diff
@@ -2,6 +2,7 @@
 L.StyleEditor.forms.GeometryForm = L.StyleEditor.forms.Form.extend({
   options: {
     formElements: {
+      'popupContent': L.StyleEditor.formElements.PopupContentElement,
```

Comment:
could you move this beneath 'size'?
I presume most users will more often change the icons and not necessarily the popupContents